hoot-core/src/main/cpp/hoot/core/io/OgrWriter.h

```diff
@@ -64,21 +64,6 @@ class OgrWriter : public PartialOsmMapWriter, public Configurable
 
   static int logWarnCount;
 
-  /**
-   * A true/false value to determine whether or not all layers are created.
-   * @sa setCreateAllLayers
-   */
-  static QString createAllLayersKey() { return "ogr.writer.create.all.layers"; }
-  /**
-   * Prepends this onto each of the layer names.
-   */
-  static QString preLayerNameKey() { return "ogr.writer.pre.layer.name"; }
-  static QString scriptKey() { return "ogr.writer.script"; }
-  /**
-   * Valid values are "on", "off" and "warn"
-   */
-  static QString strictCheckingKey() { return "ogr.strict.checking"; }
-
```

Comment:
These weren't being called anywhere and have equivalent functions in `ConfigOptions.h`